shared/deps/serial/VirtIoChannel_guest_send_receive.py

```diff
@@ -95,7 +95,9 @@ def shake_hand(self, size=0, action="receive"):
                 raise ShakeHandError("Host didn't ACK the file size message.")
             return size
         elif action == "receive":
-            txt = self.receive(hi_msg_len)
+            txt = ''
+            while len(txt) < hi_msg_len:
+                txt += self.receive(hi_msg_len)
```

Comment:
And if we have to add this waiting behavior, the same thing should be done for `serial_host_send_receive.py` as there is no guarantee of which script must be launched at first.

Reply:
@luckyh `serial_host_send_receive.py` has no such issue since it use `socket.recv` instead of `os.read`, it will wait until receive something by itself. I have tested it does wait there until receive handshake message.

Reply:
ok, thanks for the clarification.